sympy/integrals/tests/test_transforms.py

```diff
@@ -480,7 +480,7 @@ def test_laplace_transform():
 
     assert LT(exp(t), t, s)[:2] == (1/(s - 1), 1)
     assert LT(exp(2*t), t, s)[:2] == (1/(s - 2), 2)
-    assert LT(exp(a*t), t, s)[:2] == (1/(s - a), a)
+    assert LT(exp(a*t), t, s)[:2] == (1/(s - a), 0)
```

Comment:
The additional constraints are different, so it should be the same solution. I will look more into it.

Reply:
What's happening there?

Reply:
It seems like a/0 is found from the general solution constraints using patterns. With the (sometimes) new canonical forms, these patterns must be updated. Will do that.

Reply:
Turned out to be related to this, but not where I expected.